elitzur-core/src/main/scala/com/spotify/elitzur/validators/Validator.scala

```diff
@@ -145,12 +147,14 @@ private[elitzur] class OptionValidator[T: Validator] extends Validator[Option[T]
   }
 
   override def shouldValidate: Boolean = true
+  override def validationType: String = "OptionValidator"
```

Comment:
We use validationType to report metrics IIRC. I have a few thoughts about this styling
- we should not leak the name "Validator" in validation types, I think the internal ones are named after the type itself
- what is the end result of the type here? If I understand correctly we are losing some detail about the actual underlying type, right? Like if I have a Option[CaseClass] or an Option[ValidationType] what happens in that case
- I'm not even sure what users want reported here. What do you want the final report to look like?

Reply:
This was just a consequence of me moving validationType to the Validator trait. There was no real purpose to using it.